frontend/src/locale/src/es.json

```diff
@@ -640,5 +673,8 @@
 	},
 	"users": {
 		"defaultMessage": "Usuarios"
+	},
+	"auto": {
+		"defaultMessage": "Auto"
```

Comment:
looks like this file has not been sorted by the script

Reply:
Sorry, it doesn't run, I'm updating them already sorted.